### PR TITLE
made function get_attendence

### PR DESCRIPTION
**Pull request by jijinharidas**

### CODE REVIEW

**Comment by subins2000** [comment]
@jijinkh kidu

File: get.py [hunk 1]
```diff
@@ -1,40 +1,47 @@
 import requests
 import json
-
-username = input("Enter the username: ")
+username = input("Enter the username: ").upper()
 password = input("Enter the password: ")
+def get_attendence(username, password):
```
Comment (subins2000) [inline get.py]: spelling

**Review by subins2000 — CHANGES_REQUESTED** [review]
correct the spelling of attendance

**Review by jijinharidas — COMMENTED** [review]
Corrected spelling

**Pull request merged**